Deprecate `@linq` in favor of re-exporting `@chain`. (#262)
JuliaData/DataFramesMeta.jl: docs/src/index.md

end
```

> By default, `@chain` places the argument returned from the 

Reviewer: ```suggestion
By default, `@chain` places the value of the 
```
(expression is by definition something that has a value; I expecially like this explanation https://fsharpforfunandprofit.com/posts/expressions-vs-statements/#:~:text=In%20programming%20language%20terminology%2C%20an,and%20doesn't%20return%20anything.)

end
```

By default, `@chain` places the value of the 
previous expression into the first argument of the current
expression. The placeholder `_` is used to break that convention
and refer to the argument returned from the previous 
expression.

```julia
@chain df begin 
    map(_.a, _.x) do a, x
        a + x
    end

> end

Reviewer 2: This example isn't super useful in practice. Maybe show a more realistic one? E.g. one where `_` is used after a few transformations?

Author: Added a better one.

```

> `@chain` also provides the `@aside` macro-flag to perform opertaions

Reviewer 2: ```suggestion
`@chain` also provides the `@aside` macro-flag to perform operations
```

in the middle of a `@chain` block. 

```julia
@chain df begin 
    @transform y = 10 .* :x

>     @aside y_mean = mean(_.y)

Reviewer 2: Mention `@aside` somewhere? Otherwise one might think it's a DataFramesMeta macro.

Author: Added a clarifying comment.
